docs/conf.py

```diff
@@ -86,3 +86,5 @@
 redirects = {
     "reference/openmp/openmp": "../../about/compatibility/openmp.html"
 }
+
+numfig = False
```

Comment:
Disables automatic numbering of figures. We only have figures in the docs in `how-to/llm-fine-tuning-optimization/` and `tuning-guides/mi300x.rst`.

I believe @lpaoletti is on board with this?

Reply:
yup